projects/basisstatistiek.md

```diff
@@ -0,0 +1,49 @@
+---
+abstract: Descriptive statistics about the population and real estate of Amsterdam
+reusability: 1
+---
+
+# Basisstatistiek
+
+**This project in only available in Dutch**
+
+All documentation, scripts and syntaxes, and datamodels (refered to as `code`) used for generating
+descriptive statistics about the population and real estate of Amsterdam.
+
+User friendly documentation can be found on our [documentation](https://basisstatistiek.amsterdam.nl) website.
+
+The project is built modular and split in various components which are split across various (github) repositories:
```

Comment:
This is awesome :) perhaps there can be a very short description of what which is, but this is optional.

Reply:
Done